src/Raven.Client/Documents/Conventions/DocumentConventions.cs

```diff
@@ -705,6 +705,9 @@ public JsonSerializer CreateSerializer()
             jsonSerializer.Converters.Add(ParametersConverter.Instance);
             jsonSerializer.Converters.Add(JsonLinqEnumerableConverter.Instance);
             jsonSerializer.Converters.Add(JsonIMetadataDictionaryConverter.Instance);
+            jsonSerializer.Converters.Add(BlittableJsonConverter.Instance);
+            jsonSerializer.Converters.Add(LazyStringValueJsonConverter.Instance);
+            jsonSerializer.Converters.Add(StreamConverter.Instance);
```

Comment:
Not sure if we want to have these as default in client? Those are used only on server-side?

Reply:
Yes, it is better to avoid them there by default.

Reply:
done